kill-counter.lic

```diff
@@ -0,0 +1,99 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#kill-counter
+=end
+
+unless HAVE_GTK
+  respond
+  respond 'error: ruby-gtk bindings are not installed or failed to load'
+  respond
+  exit
+end
+
+custom_require.call(%w(textsubs))
+
+class KillCounter
+  def initialize
+    @kill_count = 0
+  end
+
+  def process_line(line)
+    if /^You search (?!around for a moment)/i =~ line
+      true
+    else
+      false
+    end
+  end
+
+end
+
+filter = KillCounter.new
```

Comment:
Previously I was checking based on the regex \balanced?]\i. But that wasn't catching kills from spells. So our method of detection may change int he future, but the intent is to detect kills.